geode-assembly/geode-assembly-test/build.gradle

```diff
@@ -32,5 +32,6 @@ dependencies {
   compileOnly('org.apache.logging.log4j:log4j-api')
   compileOnly('org.assertj:assertj-core')
   compileOnly('org.codehaus.cargo:cargo-core-uberjar')
+  implementation(project(':geode-logging'))
```

Comment:
Nitpicky but maybe lump this in the other `project` depedencies :)

Reply:
check.